src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/mathfuncs.txt

```diff
@@ -260,4 +260,22 @@ Blank()
 Blank()
 
 >> Int(1/0)
-#Error
\ No newline at end of file
+#Error
+
+>>IsNumeric("A")
+false
+
+>>IsNumeric(123)
+true
+
+>>IsNumeric(-123)
+true
+
+>>IsNumeric(-123.1)
+true
+
+>>IsNumeric(123.1)
+true
+
+>>IsNumeric("1" + 2)
+true
```

Comment:
Also, `IsNumeric(1/0)` and `IsNumeric(Blank())` and maybe `IsNumeric({RecordValue: 123))`, `IsNu meric([1,2,3,4])`

Reply:
I've added those as well!